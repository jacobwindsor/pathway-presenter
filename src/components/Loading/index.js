import React from 'react';
import Spinner from 'react-spinkit';
import './index.css';

export default () =>
  <Spinner
    name="three-bounce"
    color="steelblue"
    fadeIn="quarter"
    className="loading-spinner"
  />;
