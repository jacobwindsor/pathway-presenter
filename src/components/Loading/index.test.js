import React from 'react';
import { shallow } from 'enzyme';
import Loading from './index';
import toJson from 'enzyme-to-json';

it('renders without failing', () => {
  const comp = shallow(<Loading />);
  expect(toJson(comp)).toMatchSnapshot();
});
