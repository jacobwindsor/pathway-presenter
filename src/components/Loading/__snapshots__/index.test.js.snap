// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without failing 1`] = `
<Spinner
  className="loading-spinner"
  color="steelblue"
  fadeIn="quarter"
  name="three-bounce"
  noFadeIn={false}
  overrideSpinnerClassName=""
/>
`;
